source/globalCommands.py

```diff
@@ -113,7 +114,8 @@
 #: Script category for document formatting commands.
 # Translators: The name of a category of NVDA commands.
 SCRCAT_DOCUMENTFORMATTING = _("Document formatting")
-
+# Translators: Reported when trying to toggle an unsupported setting in speech emulation mode.
+speechEmulationMSG = _('Action unavailable while the braille mode is set to speech emulation')

```

Comment:
Please ensure there are appropriate new lines and double quotations are used instead of single quotations

```suggestion

# Translators: Reported when trying to toggle an unsupported setting in speech emulation mode.
speechEmulationMSG = _("Action unavailable while the braille mode is set to speech emulation")


```